docs/additional-configuration.adoc

```diff
@@ -325,7 +327,9 @@ spec:
         attributes:
           container-overrides: {"resources":{"limits":{"nvidia.com/gpu":"1"}}}
 ----
-Note that the container-overrides field does not allow configuring fields that are available through the regular DevWorkspace API. In particular, workspaces will fail to start if the container-overrides attribute attempts to override `name`, `image`, `command`, `args`, `ports`, `volumeMounts`, or `env`.
+Note that the container-overrides field does not allow configuring most fields that are available through the regular DevWorkspace API. In particular, workspaces will fail to start if the container-overrides attribute attempts to override `name`, `image`, `command`, `args`, `ports`, or `env`.
+
+Note that extra precaution should be taken when overriding the `volumeMounts` field. The DevWorkspace operator defines volumeMounts by default for config files, metadata, and credentials. To avoid unexpected behaviour, these default volumeMounts should not be overridden.
```

Comment:
The two last times "volumeMounts" is mentioned are missing backticks, e.g. volumeMounts should be `volumeMounts` for consistency.

My earlier re-wording suggestion could also be used here:

The DevWorkspace operator sets the `volumeMounts` by default for config files, metadata, and credentials. To avoid unexpected behaviour, the `volumeMounts` field should not be overridden.

Reply:
Thanks, updated